### PR TITLE
Calibrate L & R resolvers, set recommended inverter params, Update Inverter msgs

### PR DESCRIPTION
**Pull request by timbitss**
### Summary
- Calibrate left and right resolvers
- Update EEPROM files with recommended motor params from the Setting Up The Emrax Motor V1.2 pdf 
- Change right wheel direction to CW, left wheel direction to CCW (facing the motor shaft from outside)
- Update fast msgs to 10 ms and slow inverter msgs to 100 ms
- Remove unused inverter CAN msgs from dbc file. See [excel sheet](https://docs.google.com/spreadsheets/d/1I4MKK22DZTSXN9jGVQjMx2Z-H5mULfPpAcJGnoBUxyc/edit?usp=sharing)

### Testing Done
- Tested on car, motors sound/run smoother

### Checklist
*Please change `[ ]` to `[x]` when you are ready.*
- [x] I have read and followed the code conventions detailed in [README.md](../README.md) (*This will save time for both you and the reviewer!*).
- [x] If this pull request is longer then **500** lines, I have provided *explicit* justification in the summary above explaining why I *cannot* break this up into multiple pull requests (*Small PR's are faster and less painful for everyone involved!*).

### CODE REVIEW

**Comment by pasitlaothamatas** [comment]
Could you explain why we are reversing the direction for the right wheel?

**Comment by pasitlaothamatas** [comment]
Lemme know n we should be good to merge

File: inverter-eeprom-config/saved_EEPROM_configs/left_inv [hunk 1]
```diff
@@ -57,19 +57,19 @@ Coast_Lo_EEPROM_(V)_x_100             ,      0
 Coast_Hi_EEPROM_(V)_x_100             ,      4  
 Accel_Max_EEPROM_(V)_x_100            ,    490  
 Pedal_Hi_EEPROM_(V)_x_100             ,    499  
-Motor_Torque_Limit_EEPROM_(Nm)_x_10   ,    500  
+Motor_Torque_Limit_EEPROM_(Nm)_x_10   ,   1320
```
Comment (Eddy13013) [inline inverter-eeprom-config/saved_EEPROM_configs/left_inv:60]: Why do we have a torque limit set? Is this number at least much higher than we would request from the DCM?

Comment (timbitss) [reply inverter-eeprom-config/saved_EEPROM_configs/left_inv:60]: A torque limit of 132 Nm is set in the EEPROM based on the maximum current through the Emrax motor. The maximum torque request from the DCM is 90 Nm currently, so this is not a problem.

**Comment by timbitss** [comment]
> Could you explain why we are reversing the direction for the right wheel?

It's a result of calibrating the resolver. Prior to resolver calibration, sending a forward rotation request was actually rotating the motor shaft in reverse which is not what the ac motor controller expects. See Section 2.2 Verify Resolver and Motor Direction in the Resolver calibration pdf for a better explanation

**Comment by pasitlaothamatas** [comment]
> > Could you explain why we are reversing the direction for the right wheel?
> 
> It's a result of calibrating the resolver. Prior to resolver calibration, sending a forward rotation request was actually rotating the motor shaft in reverse which is not what the ac motor controller expects. See Section 2.2 Verify Resolver and Motor Direction in the Resolver calibration pdf for a better explanation

@timbitss Ok so i gave the doc a read through. The doc states that "The controller does not understand which direction for a vehicle is “Forward” and which is “Reverse”" but also mentions that "Generally for most motors the Forward direction is counter-clockwise rotation when facing the motor shaft."  (salutes to `ly for most motors the Forward direction is counter-clockwise rotation when facing the motor shaft` @kirons96  (thank you please clap))

So before calibrating, sending a positive torque request would turn the shaft counter clockwise (when facing the shaft)? After calibrating we have now defined the forward direction to be the clockwise direction?

**Comment by pasitlaothamatas** [comment]
@timbitss Do we need to update for `GenMsgCycleTime` for the Torque command, motor speed, etc. CAN messages that we receive from the inverters?

**Comment by timbitss** [comment]
> 
> So before calibrating, sending a positive torque request would turn the shaft counter clockwise (when facing the shaft)? After calibrating we have now defined the forward direction to be the clockwise direction?

After calibrating, we have now defined the forward direction to be in the **counter-clockwise** direction. 

`Direction_Command_INVR` = 0 means spin the motor in the reverse / clockwise direction which is what we want

**Comment by timbitss** [comment]
> @timbitss Do we need to update for `GenMsgCycleTime` for the Torque command, motor speed, etc. CAN messages that we receive from the inverters?

Good catch. Functionally I don't think it affects anything but yes I will update those values

**Review by pasitlaothamatas — APPROVED** [review]
LGTM

**Pull request merged**